lib/pacio_test_kit/common_tests/adi_search_tests/patient_search_tests/patient_deathdate_family_search_test.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../../searcht_test'
+
+module PacioTestKit
+  class PatientDeathdateFamilySearchTest < Inferno::Test
+    include PacioTestKit::SearchTest
+
+    title 'Server returns valid results for search by deathdate + family'
+    description %(
+      A server SHALL support searching by deathdate + family.
```

Comment:
The capability statement references [US Core Patient V7.0.0](https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-patient.html). In this US Core version, `deathdate + family` has a `SHOULD` conformance. Fix the title and mark this test as optional.